webapp/src/websocket_handlers.ts

```diff
@@ -388,8 +388,8 @@ export function handleCallHostChanged(store: Store, ev: WebSocketMessage<CallHos
         },
     });
 
-    const profiles = profilesInCurrentCallMap(store.getState());
-    const hostProfile = profiles[ev.data.hostID];
+    const hostProfile = profilesInCurrentCallMap(store.getState())[ev.data.hostID] ||
```

Comment:
This happens because we send out the host change before the join msg. I did this here bc it's less complicated than delaying the host change message server side.